lumentum.py

```
@@ -6,7 +6,7 @@
 from utils import *
```

Ruff (0.12.2)

`from utils import *` used; unable to detect undefined names

(F403)